### DISCUSSION TITLE
[Controller] Change request content for forward

### DISCUSSION THREAD

**Discussion by amigian74** [symfony/symfony#44613] [Q&A] [answered]
Hi,

I'm using the foward method of the AbstractController but now I have to change the request content. Is there any way to do this? The function itself has no parameter for this.

Thx in advance

**Comment by stof** [comment, marked as answer]
Well, use the `http_kernel` service directly if you need to do custom subrequests instead of forwarding directly. See the implementation of `AbstractController::forward`
